schemas/JSON/configuration/configuration.schema.0.1.json

```diff
@@ -0,0 +1,204 @@
+{
+    "$id": "https://aka.ms/schemas/dsc/configuration.schema.0.1.yaml",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "A representation of a configuration used by an orchestrator for WinDSC.",
+    "type": "object",
+    "properties": {
+        "assertions": {
+            "type": "array",
+            "items": { "$ref": "#/defs/resource" }
+        },
+        "resources": {
+            "type": "array",
+            "items": { "$ref": "#/defs/resource" }
+        },
+        "parameters": {
+            "type": "array",
+            "items": { "$ref": "#/defs/resource" },
+            "description": "Resources that retrieve information via a 'get' operation."
+        },
+        "configurationType": {
+            "type": "string",
+            "default": "dsc",
+            "const": "dsc",
+            "description": "The configuration type."
+        },
+        "configurationVersion": {
+            "type": "string",
+            "default": "0.1",
+            "pattern": "^(0|[1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])(\\.(0|[1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])){2}$",
+            "description": "The configuration syntax version"
+        }
+    },
+    "$defs": {
+        "resource": {
+            "type": "object",
+            "properties": {
+                "resourceName": {

```

Comment:
The proposed property here was `resource`.  Why was it changed?

Reply:
Changed to "resource". I thought having the same property name as the object could cause a naming conflict but I guess not.